static/cms/admin/config.yml

```
@@ -87,9 +87,11 @@ collections:
             label: ID
             hint: The ID of the video. For YouTube videos and shorts, this is the last part of the URL. For example, the ID of the video https://www.youtube.com/shorts/ed8A3n49Gmg is the last part ed8A3n49Gmg.
             widget: string
+            required: false
```

I think these should actually be `true`. Did you notice anything not going as expected/desired?
The `required: false` on line 83 should do what you want. These `required: true` basically assure that if you add an entry you cannot leave it empty. But you can remove the empty and don't have it at all. Although, maybe it doesn't play out as I expect it since it's an object :thinking:
If this doesn't work as it should we need to adjust a few other section for error handling.   
```suggestion
            required: true
```

---

When I tested the changes with removing the `space-between` on the action line, I wanted to specifically verify that the position of the Zip and OneClick download buttons stayed where they should without a showcase. When testing, it wouldn't allow me to post a new MOTW without these being set to false.

---

I see the issue. I guess we have no options as it seems.

https://github.com/TheCzar1994/bsaber/pull/8 should address the other needed changes to make it more robust and correct the data model.

---

Ups, kinda overlooked something: https://github.com/TheCzar1994/bsaber/pull/9